src/buildMathML.js

```diff
@@ -151,7 +151,29 @@ export const buildExpression = function(
         lastGroup = group;
```

Comment:
I think we can loop only once with less or same number of checks:
```js
            // Combine \not with mrels and mords
/*L#147*/   } else if (lastGroup instanceof MathNode && lastGroup.type === 'mi' &&
                lastGroup.children.length === 1) {
                const lastChild = lastGroup.children[0];
                if (lastChild instanceof TextNode && lastChild.text === '\u0338' &&
                    group instanceof MathNode && (group.type === 'mo' ||
                    group.type === 'mi' || group.type === 'mn')) {
                    const child = group.children[0];
                    if (child instanceof TextNode && child.text.length > 0) {
                        // Overlay with combining character long solidus
                        child.text = child.text.slice(0, 1) + "\u0338" + child.text.slice(1);
                        groups.pop();
                    }
                }
            }
```

Reply:
Done.